spec/requests/cinema_halls_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe "Movies requests" do
```

Comment:
Do you mean CityHall requests?

Reply:
Yeah, I copied all these things and changed parameters.